docs/tutorials/aws.md

```diff
@@ -23,7 +25,17 @@ Hosted Zone IDs.
       ],
       "Resource": [
         "arn:aws:route53:::hostedzone/*"
-      ]
+      ],
+      "Condition": {
+        // optional condition to further restrict from which vpc the role can be used
```

Comment:
Comments are not supported in JSON. 
The current example can be copy-pasted and ready to go.
Wdyt of adding this in an other, specific, json bloc ?

Reply:
I'm not sure if I understand correctly.

1. Need to remove comments from json
2. What exactly to add to other example, I'm not fully understand

Reply:
1. Yes, you need to remove comments. It's not possible in JSON.
2. Like for AWS SD, the [first example](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#iam-policy) can work without any edit, easier for beginners. I suggest to keep it simple and, like for AWS SD, introduce a second more secure example, that needs edits.

Reply:
Understood. Changed